internal/manager/run.go

```diff
@@ -179,7 +180,9 @@ func Run(
 	referenceIndexers := ctrlref.NewCacheIndexers(setupLog.WithName("reference-indexers"))
 	cache := store.NewCacheStores()
 	storer := store.New(cache, c.IngressClassName, logger)
-	configTranslator, err := translator.NewTranslator(logger, storer, c.KongWorkspace, translatorFeatureFlags)
+	configTranslator, err := translator.NewTranslator(logger, storer, c.KongWorkspace, translatorFeatureFlags, func() kong.AbstractSchemaService {
+		return clientsManager.GatewayClients()[0].AdminAPIClient().Schemas
```

Comment:
`GatewayClients()` might return an empty slice from https://github.com/Kong/kubernetes-ingress-controller/blob/bd686c703031c1745d98d3848c9f515d703e0ae4/internal/clients/manager.go#L181-L182 , right? In this case this would panic.

How about we create a small simple type that implements an interface `GatewayClients() []*adminapi.Client` and from that it gets the schema (as in the commented line). WDYT?

Reply:
Added an `UnavailableSchemaService` that always returns error for the case.